build/helper/metadata_add_all.py

```diff
@@ -609,7 +609,7 @@ def add_all_metadata(functions, attributes, enums, config, persist_output=True):
     fixup_enum_names(config)
 
     pp_persist = pprint.PrettyPrinter(indent=4, width=200)
-    metadata_dir = os.path.join('bin', 'processed_metadata')
+    metadata_dir = os.path.join('generated', 'processed_metadata')
```

Comment:
observation: it's too bad this is not single-sourced
Should not be fixed in this PR